rfcs/convergence/fixed-versions-for-prerelease.md

```diff
@@ -154,7 +154,22 @@ To mitigate this problem during the prerelease phase we can do the following:
 ### Prefer to release new components straight into react-components beta
 
 For sane versioning, new packages should be released straight into the `beta` prerelease tag. A new way of consuming
-unstable packages is proposed in the next section.
+unstable packages is proposed in the next section. `alpha` and `beta` are liable to breaking API changes both internally
+and externally. Having both prerelease tags existing side by side could cause dependency hell too easily.
+
+### RC release can coexist with beta components
```

Comment:
this title is not very clear / doesnt express the intent - can we rename this to something like "### Release Candidate (RC) release"

Reply:
```suggestion
### Release Candidate (RC) release"
```